gpt4all-bindings/python/gpt4all/gpt4all.py

```diff
@@ -22,7 +22,7 @@ class GPT4All():
         model: Pointer to underlying C model.
     """
 
-    def __init__(self, model_name: str, model_path: str = None, model_type: str = None, allow_download = True):
+    def __init__(self, model_name: str, model_path: str = None, model_type: str = None, allow_download = True, n_threads = None):
```

Comment:
nit: add `Optional` to the `typing` imports and make this `n_threads: Optional[int] = None`

Reply:
Hi @apage43,
thank you for your advice. I can do that. But I wonder, if this is very helptful, because some other parameters in this method definition don't use `Optional`, although they are optional and with default = None, too (`model_path: str = None, model_type: str = None>`). Isn't it confusing to mix this up in one line?

Should I change this according to your suggestion or should I leave it as it is now, because you approved the changes?

Reply:
Ah right - yeah it'd be good to fix the routing all separately from this change instead of just fixing this one - this is fine as is